python/py-pyobjc/Portfile

```diff
@@ -60,17 +63,26 @@ deployment-target = ${macosx_deployment_target}\\
         }
     }
 
+    post-patch {
+        reinplace \
+            "s/sphinx-build /sphinx-build-${python.branch} /g" \
+            ${worksrcpath}/docs/Makefile
+    }
+
+    # the pyobjc build system is vaguely horrible, so do everything in
+    # the destroot target
+    build {
+        system -W ${worksrcpath}/docs "make html"
+    }
+
+    destroot.cmd "${python.bin} install.py"
```

Comment:
Why override the default from the portgroup? This likely suffers from https://trac.macports.org/ticket/9831 as a result.

Reply:
I'll check.

Reply:
Of course you have to override because it's install.py not setup.py. But `--no-user-cfg` is probably still important.